### DISCUSSION TITLE
Question on "WAINFO" and "WACREATEKEY"

### DISCUSSION THREAD

**Discussion by ghost** [ElDavoo/wa-crypt-tools#124] [General]
I've been able to decrypt my crypt15 backup file using my 64 digit key (for crypt15). So far so good - a very well-produced and sophisticated tool! Thanks a lot.

However:
I have no idea what to enter for WACREATEKEY or WAINFO (for crypt14).

Thanks a lot!

**Comment by ElDavoo** [comment]
> a very well-produced

well, your following statements say the actual truth, the project is still a little bit raw. :)  Most notably, with encryption the combinations of possible things exploded and it's very difficult to test everything.


> I have no idea what to enter for WACREATEKEY or WAINFO (for crypt14).

They still don't support crypt14, but you can use the universal option `-h` or `--help` to get the usage information.
